### PR TITLE
My porfolio desktop version

### PR DESCRIPTION
**Pull request by nimplay**
Summary:

This is the desktop version of my portfolio. Full response made with media query

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -806,11 +788,321 @@ header {
...
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 10px 12px;
+    gap: 10px;
+    width: 15%;
+    height: 40px;
+    flex: none;
+    flex-grow: 0;
+  }
```
Comment (phelian23) [inline style.css]: - [ ] Kindly add some margins between the nav links, presently, they are overlapping each other.
Your design
Capture

File: style.css [hunk 2]
```diff
@@ -806,11 +788,321 @@ header {
...
+    width: 70%;
+    order: 1;
+  }
+
+  #wintro {
+    flex-direction: row;
+    padding: 0;
+    gap: 24px;
+  }
+
```
Comment (phelian23) [inline style.css]: - [ ] Kindly make sure that the width and margins of the element on the page match that of the design template as much as possible.
Your design
Capture1
Capture2

Figma design
figma

File: style.css [hunk 3]
```diff
@@ -806,11 +788,321 @@ header {
...
+  .contactform {
+    width: 70%;
+    align-items: stretch;
+  }
+
+  .cdata {
+    width: 100%;
+  }
+
```
Comment (phelian23) [inline style.css]: - [ ] Kindly adjust the with of the text half of the contact section to match that of the design template. it should take about 35% of the page. also, the font weight should be bold. Kindly fix it.
Your design
Capture3

**Review by phelian23 — CHANGES_REQUESTED** [review]
Hi @nimplay, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Required Changes ♻️

- [ ] When adding margins, widths, heights, and padding to your page, always favor units that are percent based (%, vw, vh) over fixed unit (px). This will improve the responsiveness of your page.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can make your code better._

You can also consider:

- [ ] [Optional] The mobile version has a horizontal scroll. Kindly remove it as there should be no horizontal scroll in the mobile version.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lordkaito — APPROVED** [review]
Hi @nimplay, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**